src/Map/src/Bridge/Google/assets/src/map_controller.ts

```diff
@@ -297,6 +302,24 @@ export default class extends AbstractMapController<
         return content;
     }
 
+    protected doCreateIcon({ definition, element }): void {
+        const { content, iconType, width, height } = definition;
+        if (iconType === 'html') {
+            const parser = new DOMParser();
```

Comment:
If we have 40 html icons, we don't want to initialize a `DOMParser` each time, please move it somewhere else 🙏🏻